src/App.test.tsx

```diff
@@ -41,6 +41,7 @@ describe('App', () => {
     const { store } = await act(async () => renderWithProviders(<App />));
 
     const classes = store.getState().class;
+
```

Comment:
This would probably be better to do in the next PR where tests are getting substantial changes https://github.com/City-of-Helsinki/infraohjelmointi-ui/pull/41/checks?check_run_id=12751774186.